packages/ai-jsx/src/batteries/docs.test.ts

```diff
@@ -1,3 +1,132 @@
-test('adds 1 + 2 to equal 3', () => {
-  expect(1 + 2).toBe(3);
+import { describe, expect, test } from '@jest/globals';
```

Comment:
We shouldn't need to do this. Jest tests are typically written with the presumption that these globals are in scope.

Reply:
See https://github.com/Microsoft/TypeScript/issues/31226

I'm open to other ways of doing this, but essentially it seems we have to choose between also distributing our tests (and getting them to compile and run from dist) or adding this import.  I don't think adding the import is really a big deal so I opted for not needlessly increasing our distributed binary size.

Reply:
We definitely don't want to distribute the tests, so if this import is the way to avoid that, I'm fine with it.

I'm slightly surprised that we'd need this import to avoid that, but :shrug:. :smile: